lib/ts/recipe/multifactorauth/constants.ts

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
```

Comment:
all copy right should say 2024 for new files

Reply:
Done